### PR TITLE
6121-Staff manually suspend account

### PR DESCRIPTION
**Pull request by karthik-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/6121

*Description of changes:*

- Added Account Status and Suspend Account button in AccountInfo vue component, with constraint that would be visible only to staff role.
- Open dialog that shows Suspend/ Unsuspend confirmation on suspend account event click.
- Added user and admin Account Suspended notification screens
- Added test cases for the same

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/views/auth/account-freeze/AccountSuspendedView.vue [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component({
+  props: {
```
Comment (shabeeb-aot) [inline auth-web/src/views/auth/account-freeze/AccountSuspendedView.vue:24]: @Prop({ default: false }) isAdmin: boolean would be good

**Review by shabeeb-aot — APPROVED** [review]

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue [hunk 2]
```diff
@@ -176,7 +241,10 @@ export default class AccountInfo extends Mixins(AccountChangeMixin) {
   private readonly currentOrganization!: Organization
   private readonly currentOrgAddress!: Address
   private readonly currentMembership!: Member
+  private readonly currentUser!: KCUserProfile
   private readonly permissions!: string[]
+  private dialogTitle = ''
```
[the remark this answers is not in the document]
Comment (karthik-aot) [reply auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue]: Refactored

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue [hunk 3]
```diff
@@ -194,12 +263,21 @@ export default class AccountInfo extends Mixins(AccountChangeMixin) {
...
   private isFormValid (): boolean {
     return !!this.orgName || this.orgName === this.currentOrganization?.name
   }
 
+  private get isStaff (): boolean {
+    return this.currentUser.roles.includes(Role.Staff)
+  }
+
+  private isSuspendButtonVisible (): boolean {
+    return this.currentOrganization.statusCode === AccountStatus.ACTIVE || this.currentOrganization.statusCode === AccountStatus.SUSPENDED
```
Comment (saravanpa-aot) [inline auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:278]: Hm..I think the logic is other wise ; If statusCode != NSF_SUSPENDED ; show the button ..we have other statuses as well..

File: auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue [hunk 4]
```diff
@@ -382,12 +475,29 @@ export default class AccountInfo extends Mixins(AccountChangeMixin) {
     return false
   }
 
+  get isAccountStatusActive () : boolean {
+    return this.currentOrganization.statusCode === AccountStatus.ACTIVE
```
Comment (saravanpa-aot) [inline auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue:479]: Again ; the logic is show UNSUSPEND if status is SUSPENDED..  for all other status , we need to show suspend..we have other statuses like PENDING_AFFIDAVIT_REVIEW for which we should show suspend button

**Pull request merged**